site/docs/pages/index.mdx

```diff
@@ -103,7 +89,10 @@ bun add @coinbase/onchainkit
   schemaId="0xf8b05c79f090979bf4a80270aba232dff11a10d9ca55c4f88de95317970f0de9"
 >
   <div className="flex h-10 items-center space-x-4">
-    <Avatar address="0x838aD0EAE54F99F1926dA7C3b6bFbF617389B4D9" showAttestation /> // [!code focus]
+    <Avatar
+      address="0x838aD0EAE54F99F1926dA7C3b6bFbF617389B4D9"
+      showAttestation
+    /> // [!code focus]
```

Comment:
When formatting the code that has `// code focus` the experience will be changed. If you check https://onchainkit.xyz/ now this line is not anymore correctly highlithed.

Which means we introduced a UI regression.